cloudevents/SDK.md

```diff
@@ -141,3 +141,56 @@ Each SDK must provide examples using at least HTTP transport of:
 - Composing an Event.
 - Encoding and sending a composed Event.
 - Receiving and decoding an Event.
+
+## Feature Support
+
+Each SDK must update the following "support table" periodically to ensure
+they accurately the status of each SDK's support for the stated features.
+
+| Feature | C#  | Go  | Java | JS  | PHP | PS  | Python | Ruby | Rust |
+| :------ | :-: | :-: | :--: | :-: | :-: | :-: | :----: | :--: | :--: |
+| **[v1.0](https://github.com/cloudevents/spec/tree/v1.0)** |
+| [CloudEvents Core](https://github.com/cloudevents/spec/blob/v1.0/spec.md) | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| |
+| [Avro Event Format](https://github.com/cloudevents/spec/blob/main/cloudevents/formats/avro-format.md) | | | | :x: | | | | | | |
+| [AMQP Event Format - need link]() | | | | :x: | | | | | | |
+| [JSON Event Format](https://github.com/cloudevents/spec/blob/main/cloudevents/formats/json-format.md) | | | | :heavy_check_mark: | | | | | | |
+| [Protobuf Event Format](https://github.com/cloudevents/spec/blob/main/cloudevents/formats/protobuf-format.md) | | | | :x: | | | | | | |
+||
+| [AMQP Binding](https://github.com/cloudevents/spec/blob/main/cloudevents/bindings/amqp-protocol-binding.md) | | | | :x: | | | | | | |
+| [HTTP Binding](https://github.com/cloudevents/spec/blob/main/cloudevents/bindings/http-protocol-binding.md) | | | | :heavy_check_mark: | | | | | | |
+| [Kafka Binding](https://github.com/cloudevents/spec/blob/main/cloudevents/bindings/kafka-protocol-binding.md) | | | | :heavy_check_mark: | | | | | | |
+| [MQTT Binding](https://github.com/cloudevents/spec/blob/main/cloudevents/bindings/mqtt-protocol-binding.md) | | | | :x: | | | | | | |
+| [NATS Binding](https://github.com/cloudevents/spec/blob/main/cloudevents/bindings/nats-protocol-binding.md) | | | | :x: | | | | | | |
+| [WebSockets Binding](https://github.com/cloudevents/spec/blob/main/cloudevents/bindings/websockets-protocol-binding.md) | | | | :x: | | | | | | |
+| |
+| [HTTP Content Mode Binary](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md) | | | | :heavy_check_mark: | | | | | | |
+| [HTTP Content Mode Structured](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md) | | | | :heavy_check_mark: | | | | | | |
+| [HTTP Content Mode Batch](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md) | | | | :heavy_check_mark: | | | | | | |
+| [Kafka Content Mode Structured]() | | | | :heavy_check_mark: | | | | | | |
+| [Kafka Content Mode Binary]() | | | | :heavy_check_mark: | | | | | | |
+| [Kafka Content Mode Structured]() | | | | :heavy_check_mark: | | | | | | |
```

Comment:
Duplicate Kafka Content Mode Structured, I guess one is meant to be "batch"?

Reply:
Except that the Kafka binding doesn't define support a batch mode...